packages/react/src/ReactRenderUtil.ts

```diff
@@ -20,13 +20,12 @@ export function renderToDOMSpec(
   }
 
   // clone so we can unmount the react root
-  contentDOM?.setAttribute("data-tmp-find", "true");
+  contentDOM?.setAttribute("data-editable", "");
```

Comment:
Although this change is possible, maybe it's nicer to keep concerns separate and keep the old implementation?

The responsibility for data-editable is then completely somewhere else, instead of in two locations. Unless removing it here and reverting to data-tmp-find breaks anything?